companion-app/src/Components/UploadFile.js

```diff
@@ -35,14 +46,32 @@ class UploadFile extends Component {
 
         this.state = {
           Filename: "",
-          playValue: 'false',
-          volume: 0.0
+          playValue: "true",
+          stop: "false",
+          volume: 0.0,
+          eqapply: "false",
+          eqlowgain: 0.0,
+          eqmidgain: 0.0,
+          eqhighgain: 0.0,
+          echoapply: "false",
+          echodelay: 10.0,
+          echofeedback: 0.0,
+          echodry: 0.0,
+          echowet: 0.0
         };
```

Comment:
Make sure that each value is sent in as a only strings

Reply:
Also if we are going to use floats, I will need to change the packet validation